src/Symfony/Bundle/FrameworkBundle/Controller/RedirectController.php

```diff
@@ -157,4 +157,23 @@ public function urlRedirectAction(Request $request, string $path, bool $permanen
 
         return new RedirectResponse($url, $statusCode);
     }
+
+    public function __invoke(Request $request): Response
+    {
+        $p = $request->attributes->get('_route_params', []);
+
+        if (\array_key_exists('route', $p)) {
+            if (\array_key_exists('path', $p)) {
+                throw new \RuntimeException('Ambiguous redirection settings, use the "path" or "route" parameter, not both.');
```

Comment:
`Ambiguous redirection settings, use the "path" or "route" parameter, not both: "%s" and "%s" found respectively.`

Reply:
even better would be to include the name of the current route (`$request->attributes->get('_route')`) instead, as that's the place where you will need to make the fix. It will be more efficient that searching by values of the path and route to find where you used that combination.

Reply:
both comments addressed.